pkg/protoassert/assert.go

```diff
@@ -40,6 +40,17 @@ func SlicesEqual[T proto.Message](t testing.TB, expected, actual []T, msgAndArgs
 	return areEqual
 }
 
+func SliceContains[T proto.Message](t testing.TB, slice []T, element T, msgAndArgs ...interface{}) bool {
+	t.Helper()
+	for _, e := range slice {
+		if proto.Equal(e, element) {
+			return true
+		}
+	}
+	t.Fatalf("Slice does not contain element: %s %v", element.String(), msgAndArgs)
```

Comment:
`t.Fatalf` will interrupt the test execution (more or less the `require` variant of an assertion).
Do we want that or should `t.Errorf` (non-interrupting variant) be called instead ?

Reply:
Good catch, I fixed that with `assert.Fail`